### ISSUE TITLE
Survival Score issue

### ISSUE THREAD

**Issue by 594872799** [robocode-dev/tank-royale#122]
In the scoring rules,  the description states: "Each bot that survives a round scores 50 points every time another bot dies." But my debugging results don’t match this. For example, how could two tanks in two rounds have a survival score of 150? 

![image](https://github.com/user-attachments/assets/5f61928c-b73c-4de9-b12f-55150b77e4a7)

https://robocode-dev.github.io/tank-royale/articles/scoring.html

**Comment by 594872799** [comment]
@flemming-n-larsen Please help me take a look.

**Comment by 594872799** [comment]
It seems like this will multiply by the number of rounds.

**Comment by flemming-n-larsen** [comment]
@594872799
Sorry, for the late reply. It is Christmas in Denmark.

I will have a look into this. 🙂

**Comment by flemming-n-larsen** [comment]
@594872799 
You are right. After two rounds, the Survival Score should be 2 x 50 = 100, and the Last Survival Bonus should be 2 x 10 = 20

I will fix this.

**Comment by 594872799** [comment]
@flemming-n-larsen I’m sorry to bother you, but I would like to know when this issue is expected to be fixed, as we are preparing for a competition, and this score has a significant impact on us.

**Comment by flemming-n-larsen** [comment]
This bug has been fixed with release [0.28.1](https://github.com/robocode-dev/tank-royale/releases/tag/v0.28.1)
